doc/using-logging.rst

```diff
@@ -7,26 +7,33 @@ framework. ZConfig provides one simple convenience function to do this:
 
 .. autofunction:: ZConfig.configureLoggers
 
-
 Suppose we have the following logging configuration in a file called ``simple-root-config.conf``:
 
 .. literalinclude:: simple-root-config.conf
```

Comment:
Yeah, things look good on RTD, even before this: http://zconfig.readthedocs.io/en/latest/using-logging.html

The doctest examples were small enough that the prompts weren't much of an eyesore, at least not to me.